src/react-themer/index.js

```diff
@@ -33,17 +34,23 @@ export default (customThemer: ?Object) => (theme?: Object) => (component: React.
   let resolvedAttrs;
   let themesToResolve;
 
-  return class extends Component {
+  class Main extends Component {
     static displayName = `Themer(${getDisplayName(rawThemerAttrs.component) || 'Component'})`;
     static rawThemerAttrs = rawThemerAttrs;
 
+    static PropTypes = {};
+
     static contextTypes = {
       theme: PropTypes.object,
     };
 
     componentWillMount() {
       const { theme: globalTheme } = this.context;
 
+      each(themerInstance.getVariants(), (value, key) => {
+        Main.PropTypes[key] = React.PropTypes.bool;
+      });
```

Comment:
I don't necessarily see this as a requirement, but I think it's probably worth doing